srv/modules/runners/deepsea.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+This runner implements helper functions for DeepSea in general
+"""
+
+from __future__ import absolute_import
+
+import os
+import re
+
+
+def version():
```

Comment:
Should be `def version(**kwargs):`.

Without `**kwargs` this runner cannot be called from the **salt-api**.

Reply:
done!